carAPI/src/main/java/com/everis/control/CarService.java

```diff
@@ -0,0 +1,56 @@
+package com.everis.control;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+import com.everis.entity.Car;
+
+public class CarService {
+	
+	private static Map<Long, Car> cars = new HashMap<Long, Car>();
+	
+	public CarService() {
+		// TODO Auto-generated constructor stub
```

Comment:
It is good to have TODO comments as a reminder of something "to do", but if it is an auto generated comment then better to remove it.